inc/sairoute.h

```diff
@@ -67,16 +67,23 @@ typedef enum _sai_route_entry_attr_t
      * @brief Next hop or next hop group id for the packet, or a router interface
      * in case of directly reachable route, or the CPU port in case of IP2ME route
      *
+     * The next hop id is only effective when the packet action is one of the following:
+     *  FORWARD, COPY, LOG, TRANSIT
+     *
      * The next hop id can be a generic next hop object, such as next hop, next
      * hop group. Directly reachable routes are the IP subnets that are
      * directly attached to the router. For such routes, fill the router
      * interface id to which the subnet is attached. IP2ME route adds a local
      * router IP address. For such routes, fill the CPU port
      * (#SAI_SWITCH_ATTR_CPU_PORT).
      *
+     * When it is SAI_NULL_OBJECT_ID, then packet will be dropped.
```

Comment:
Just to confirm the understanding that NULL_OBJECT_ID can be set when packet action is set to TRAP before and the packet action will take precedence in that case.  In other words, the packet will be dropped when packet action is one of the following: FORWARD, COPY, LOG, TRANSIT as mentioned in the comments.